a3p-integration/proposals/a:upgrade-next/upgradeVaults.test.js

```diff
@@ -95,6 +96,15 @@ const checkAuctionVat = async t => {
   t.true(Object.keys(details).length > 2);
 };
 
+const verifyVaultPriceUpdate = async t => {
+  const quote = await getVaultPrices(0);
+
+  t.true(quote.value[0].amountIn.brand.includes(' ATOM '));
+
+  // FAIL, should be +5200000
+  t.is(quote.value[0].amountOut.value, '12010000');
```

Comment:
> Nit I prefer seeing tests annotated with .failing but that's ok in this case.

Yes, I wanted to, but I didn't want to mark the entire sequence of tests as failing, just the one test case.

Multiple test cases are bundled into one `test()` call because they have to be executed in order and `test.serial()` doesn't guarantee order.